5calls/app/src/main/java/org/a5calls/android/a5calls/FiveCallsApplication.java

```diff
@@ -18,13 +18,18 @@
 
 import android.app.Activity;
 import android.app.Application;
+import android.app.NotificationManager;
```

Comment:
looks like maybe this one isn't used? (I don't get android studio's helpful highlighting from github)